resources/views/layouts/default.blade.php

```diff
@@ -74,7 +74,8 @@
           };
     </script>
     <!-- Add laravel routes into javascript  Primarily useful for vue.-->
-    @routes
+    {{-- @routes --}}
```

Comment:
This bit is *definitely* a problem. But if we leave it there I can't load up any blades.

Reply:
I 'fixed' it by just yanking Ziggy entirely... which makes this change much bigger than I would've wanted :(

Reply:
I know it’s bigger than you wanted, but if we can get it right, I will be THRILLED to lose that package.